apps/prairielearn/src/tests/sync/assessmentsSync.test.ts

```diff
@@ -2011,39 +1982,6 @@ describe('Assessment syncing', () => {
     assert.equal(syncedAssessment.sync_errors, null);
   });
 
-  it('records an error if multiple-element points array is specified for an alternative when real-time grading is disallowed', async () => {
```

Comment:
Just confirming: these two 'deleted' tests still exist/functionality was replicated further down, they were just marked in the diff incorrectly?